Source/WebKit/UIProcess/ios/forms/WKDateTimeInputControl.mm

```diff
@@ -67,31 +77,48 @@ @implementation WKDateTimePicker
 static NSString * const kMonthFormatString = @"yyyy-MM"; // "2011-01".
 static NSString * const kTimeFormatString = @"HH:mm"; // "13:45".
 static NSString * const kDateTimeFormatString = @"yyyy-MM-dd'T'HH:mm"; // "2011-01-27T13:45"
+// Weekday is necessary for week inputs in order to append monday to the initial value so that the NSDate reflects weeks beginning on Mondays in ISO-8601
```

Comment:
```suggestion
// Weekday is necessary for week inputs in order to append Monday to the initial value so that the NSDate reflects weeks beginning on Mondays in ISO-8601.
```